debian/securedrop-workstation-config.securedrop-mime-handling.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Securedrop Mimetype Handling Override
+ConditionPathExists=/var/run/qubes-service/securedrop-mime-handling
+ConditionPathExists=/var/run/qubes/persistent-none
```

Comment:
This is causing it to not run for sd-app. We can move this condition to the script for now?